teku/src/main/java/tech/pegasys/teku/cli/options/BeaconRestApiOptions.java

```diff
@@ -130,6 +130,16 @@ public void setMaxUrlLength(int maxUrlLength) {
   private int validatorThreads = BeaconRestApiConfig.DEFAULT_SUBSCRIBE_THREADS_COUNT;
 
   public void configure(final TekuConfiguration.Builder builder) {
+    // Set defaults
+    if (restApiEnabled == null && restApiPort == null) {
+      restApiEnabled = false;
+      restApiPort = BeaconRestApiConfig.DEFAULT_REST_API_PORT;
+    } else if (restApiEnabled == null) {
+      restApiEnabled = true;
+    } else if (restApiPort == null) {
+      restApiPort = BeaconRestApiConfig.DEFAULT_REST_API_PORT;
+    }
+
```

Comment:
we only have 2 flags here, its perfect for testing literally every option...

restApiEnabled | restApiPort | enabled outcome | port outcome |
------------- | -- | -- | -- | 
null | null | false | UNDEF |
null | 5051 | true | 5051 |
true | null | true | 5051 |
true | 5055 | true | 5055 |

this is a really small set of test permutations, could potentially add port is set to a negative just to do some negative probing...

This is my understanding of what we want, but either way shows the idea...